docs/widgets/media-server.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 🎞️ Media Server
+tags:
+  - Widgets
+  - Integration
+  - Configuration
+  - Media Server
+  - Plex
+  - Jellyfin
+  - Sessions
+  - API
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The media server widget enables you to watch and view current active sessions over all your media servers.
+It also displays useful information, that you can use when providing support to users or debugging a problem.
+
+---
+
+## Adding the widget
+Please check out our documentation on [how to add a widget](/docs/introduction/after-the-installation#adding-widgets).
+
+---
```

Comment:
Please add some form of "Configuring" section here. Just like all other widget docs.

Reply:
Why? There is no configuration for this widget

Reply:
All widget docs have a configuration section, even if there aren't really things to configure.